xonsh/tools.py

```diff
@@ -1686,15 +1686,29 @@ def globpath(s, ignore_case=False, return_empty=False):
 
 def _iglobpath(s, ignore_case=False):
```

Comment:
Would it be an idea to add sorted kwarg to the function to override the environment variable e.g `sorted=None`. If `sorted` were `True` or `False` it would override the environment variable.